Extensions/pokes.js

```diff
@@ -25,6 +26,31 @@ XKit.extensions.pokes = {
 		XKit.extensions.pokes.checkEligibility();
 	},
 
+	fetch_pokedex: function(callback) {
+		if (!XKit.extensions.pokes.gist_cache) {
+			GM_xmlhttpRequest({
+				method: "GET",
+				url: XKit.extensions.pokes.pokedex_url,
+				json: true,
+				onerror: function(response) {
+					console.log("Poke data could not be retrieved. Skipping instance.");
```

Comment:
needs an onerror for the "failed to load pokemon data" case (where it needs to go to the cpanel.

(this is why I suggested using a promise, among other things.